src/bundler/bundle_v2.zig

```diff
@@ -4862,7 +4862,14 @@ pub const LinkerContext = struct {
                 // does it have a JS AST?
                 if (!(id < import_records_list.len)) continue;
 
+                const tla_keywords = this.parse_graph.ast.items(.top_level_await_keyword);
```

Comment:
We are now calling .items for every reachable file. This is an improvement over calling .items for every reacahble file for every impot record for every import record's record, but we can still move this one level up. It might even already be there.

Reply:
oops missed that

Reply:
https://github.com/oven-sh/bun/pull/14218/commits/bb81cd03415c042ec33aa0ccdb0019acc1fbb591